<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Poker</class>
 <widget class="QMainWindow" name="Poker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Poker</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="labelA">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelB">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>80</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>B</string>
    </property>
   </widget>
   <widget class="QLabel" name="iCall">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>240</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>叫地主</string>
    </property>
   </widget>
   <widget class="QLabel" name="iDontCall">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>210</y>
      <width>87</width>
      <height>87</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>不叫</string>
    </property>
   </widget>
   <widget class="QLabel" name="ADontCall">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>87</width>
      <height>87</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>不叫</string>
    </property>
   </widget>
   <widget class="QLabel" name="BDontCall">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>50</y>
      <width>87</width>
      <height>87</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>不叫</string>
    </property>
   </widget>
   <widget class="QLabel" name="BCall">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>80</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>叫地主</string>
    </property>
   </widget>
   <widget class="QLabel" name="ACall">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>80</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>叫地主</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>260</y>
      <width>289</width>
      <height>89</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="callButton">
       <property name="text">
        <string>叫地主</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="notcallButton">
       <property name="text">
        <string>不叫</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
